### ISSUE TITLE
how to use grid search？

### ISSUE THREAD

**Issue by qqqq7411** [junyanz/pytorch-CycleGAN-and-pix2pix#191]

**Comment by junyanz** [comment]
What do you mean "grid search"? It is not included in this repo.

**Comment by qqqq7411** [comment]
@junyanz  Thanks for your reply.I want to use grid search to optimize the hyper-parameters.But I don't know if this will work in pix2pix.

**Comment by junyanz** [comment]
I see. It might be hard to apply grid search to GANs models as often times, there is no standard metric to look at it. See this [post](https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix/issues/166) for more details.

**Comment by qqqq7411** [comment]
How about PBT？

**Comment by SsnL** [comment]
@qqqq7411 if you look at the PBT paper, they use Inception score to do param search for GANs. With CycleGAN / pix2pix, it is not necessarily true that one side is realistic photos.
